src/templates/static/legal/metatrader/biel.jsx

```diff
@@ -2,200 +2,122 @@ import React from 'react';
-                        <p>{it.L('Negative Balance Protection does not apply to clients categorized as professional traders, who can still lose more money than their available balance.')}</p>
-                        <p>{it.L('The Company does not offer credit to its clients. You acknowledge and agree that you will not be dealing with us on credit.')}</p>
+                        <p>{it.L('When using the MT5 trading platform, all BIEL\'s clients will be provided with Margin monitoring functionality to protect the Clients from encountering negative balances when trading under normal market conditions. As a trader, the Client should always maintain the appropriate levels of Margin in their trading account as the recommended method of their own risk management.')}</p>
+                        <p>{it.L('If the Client\'s usable Margin drops below 100%, a Margin Call mode will be triggered and maintained till the level of 50%. In the event that the Client\'s Margin Level is equal to, or drops below 50%, the Company will initiate the closing of the Client\'s current Open Positions, starting from the most unprofitable, until the required Margin Level is achieved. In such events, the positions will be automatically closed at the current market price at that point in time.')}</p>
+                        <p>{it.L('In exceptional circumstances, where there is a price change in the underlying that is sufficiently large and sudden, gapping can occur. In such cases, the automatic Margin close-out protection might fail, causing the Client\'s Account Balance to fall below zero. For this purpose, a Negative Balance Protection mechanism has been introduced. Negative Balance Protection provides a ‘backstop\' in case of extreme market conditions and ensures that the Client\'s maximum losses from trading CFDs, including all related costs, are limited to the balance available on the Client\'s CFD account. Therefore, the Client can never lose more money than the total sum invested for trading CFDs; if the Client\'s CFD account balance falls below zero, the Company will compensate the negative balance as soon as possible without any additional cost to the Client.')}</p>
```

Comment:
```suggestion
                        <p>{it.L('In exceptional circumstances, where there is a price change in the underlying that is sufficiently large and sudden, gapping can occur. In such cases, the automatic Margin close-out protection might fail, causing the Client\'s Account Balance to fall below zero. For this purpose, a Negative Balance Protection mechanism has been introduced. Negative Balance Protection provides a \'backstop\' in case of extreme market conditions and ensures that the Client\'s maximum losses from trading CFDs, including all related costs, are limited to the balance available on the Client\'s CFD account. Therefore, the Client can never lose more money than the total sum invested for trading CFDs; if the Client\'s CFD account balance falls below zero, the Company will compensate the negative balance as soon as possible without any additional cost to the Client.')}</p>
```